### PR TITLE
feat: Get project ready for submission

### PR DESCRIPTION
**Pull request by HarryET**

### CODE REVIEW

**Comment by HarryET** [comment]
@VictorPeralta or @Olyno, can you please review and fill any question marks and/or missing sections.

File: README.md [hunk 1]
```diff
@@ -3,3 +3,53 @@
...
+
+## Our Team
+
+| <img src="https://github.com/harryet.png" width="96" height="96"/> | <img src="https://github.com/olyno.png" width="96" height="96"/> | <img src="https://github.com/VictorPeralta.png" width="96" height="96"/> |
+|--------------------------------------------------------------------|------------------------------------------------------------------|--------------------------------------------------------------------------|
+| <a href="https://github.com/HarryET">HarryET</a>                   | <a href="https://github.com/Olyno">Olyno</a>                     | <a href="https://github.com/VictorPeralta">VictorPeralta</a>             |
+| <a href="https://twitter.com/TheHarryET">@TheHarryET</a>           | <a href="https://twitter.com/OlynoWorker">@OlynoWorker</a>       | <a href="https://twitter.com/PeraltaDev">@PeraltaDev</a>                 |
+
+## License
+Copyright (C) 2022 Harry Bairstow, Victor Peralta, Olyno
```
Comment (HarryET) [inline README.md:55]: @Olyno do you want your real name here?

Comment (Olyno) [reply README.md:55]: Olyno is good for now

**Comment by Olyno** [comment]
I can't explain the database part, so i let @VictorPeralta do it [here](https://github.com/feedbackcentral/feedbackcentral/pull/15/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R24).

File: README.md [hunk 2]
```diff
@@ -8,7 +8,19 @@ The AI Powered Feedback Aggregator for the Supabase 2022 Hackathon
 The project is available at [feedbackcentral.io](https://feedbackcentral.io), any pull request also has an automatically generated preview URL.
 
 ## How to use the project?
-For Olyno/Victor to fill out
+
```
Comment (HarryET) [inline README.md:11]: @Olyno This was more how to use the website, how to setup a project etc

Comment (Olyno) [reply README.md:11]: Is it really needed? Like if we have a landing page, we just have to login and follow a simple website logic right? Doesn't make sense to me to explain how the website work.

**Pull request merged**